<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnaGiris</class>
 <widget class="QMainWindow" name="AnaGiris">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_grs">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Giriş Yöntemi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_grs">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Giriş</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_grs">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Yönetici</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Öğrenci</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>141</width>
      <height>53</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="ln_kll"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="ln_sif"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>129</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_kll">
       <property name="text">
        <string>Kullanıcı adı:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_sif">
       <property name="text">
        <string>Şifre :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl_hata">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>368</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
